flow/Net2.actor.cpp

```diff
@@ -981,11 +995,10 @@ void Net2::initTLS() {
 			.detail("VerifyPeers", boost::algorithm::join(loaded.getVerifyPeers(), "|"));
 		ConfigureSSLContext( tlsConfig.loadSync(), &newContext, onPolicyFailure );
 		sslContextVar.set(ReferencedObject<boost::asio::ssl::context>::from(std::move(newContext)));
-		backgroundCertRefresh = reloadCertificatesOnChange( tlsConfig, onPolicyFailure, &sslContextVar );
 	} catch (Error& e) {
 		TraceEvent("Net2TLSInitError").error(e);
-		throw tls_error();
```

Comment:
Just to make sure I understand -- this change means that a failure to initialize in a non-TLS cluster will just log this event and then connect fine. Failure to initialize in a TLS cluster will log the event and keep running, but not connect to the cluster. Is that correct?

Are all of the possible initialization failures here such that they can be resolved by fixing the certificates and reloading, or are there some that will leave us permanently unable to connect? Is there any situation where a failure here leaves something in an uninitialized state that we later use and cause undefined behavior?

Reply:
That is correct, and in testing, appears to be what 6.2.15 did.  I'm honestly not sure that I agree that is the best thing to do, but I do see the value in maintaining semantics within a patch release.

In the tests that I ran through locally, I could fix everything via certificate refreshing.  However, do note that the default certificate refresh interval is once every 24 hours, so we're not going to notice certificate fixes anytime soon.  (This also is the behavior of 6.2.15)